bin/near

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env node --experimental-repl-await
```

Comment:
what's --experimental-repl-await?

Reply:
(would be helpful to leave a comment)

Reply:
seems to be very googlable, so probably not worth it:

https://www.google.com/search?client=safari&rls=en&q=--experimental-repl-await&ie=UTF-8&oe=UTF-8